src/components/home/navbar/DropdownButton.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+
+import DropdownMenu from './DropdownMenu'
+
+import '../../../styles/home/navbar/dropdown_button.css'
+
+class DropdownButton extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isOpen: false
+    }
+    this.toggleMenu = this.toggleMenu.bind(this)
+    this.onClose = this.onClose.bind(this)
+  }
+
+  toggleMenu (event) {
+    // event.stopPropagation()
+    this.setState({isOpen: !this.state.isOpen})
+  }
+
+  onClose () {
+    this.setState({isOpen: false})
+  }
+
+  componentDidMount () {
+    document.body.addEventListener('click', this.onClose)
+  }
+
+  componentWillUnmount () {
+    document.body.removeEventListener('click', this.onClose)
+  }
+
+  render () {
+    return <div>
```

Comment:
I prefer that we wrap the returned JSX in parens to provide a visual delineation of what we are rendering:
```jsx
return (
  <div>
    <!-- etc... -->
  </div>
)
```